test/export/test_export.py

```diff
@@ -6894,17 +6884,6 @@ def test(ep):
             # running decompositions again should work for all IRs
             ep = export(M(), inp, preserve_module_call_signature=("n",))
             test(ep.run_decompositions({}))
```

Comment:
previously this would test re-running decompositions, now it won't, right?

Reply:
Ahh you are right we are still decomp-ing in training IR case, i will include the code i deleted here.